### PR TITLE
More android stuff (Rocket server, output to logcat redirection)

### PR DESCRIPTION
**Pull request by ErikBjare**

### CODE REVIEW

**Comment by johan-bjareholt** [comment]
Tests are broken after endpoints parameter change

**Comment by ErikBjare** [comment]
> Tests are broken after endpoints parameter change

Fixed in https://github.com/ActivityWatch/aw-server-rust/pull/11/commits/adb915ea3570db80ddfca76f9d5e0202628d77d5

File: src/android.rs [hunk 1]
```diff
@@ -105,7 +150,101 @@ pub mod android {
...
+    // From: https://github.com/servo/servo/pull/21812/files
+    // With modifications from latest: https://github.com/servo/servo/blob/5de6d87c97050db35cfb0a575e14b4d9b6207ac5/ports/libsimpleservo/jniapi/src/lib.rs#L480
+    use std::os::raw::{c_char, c_int};
+    use std::thread;
+
+    extern "C" {
+        pub fn __android_log_write(prio: c_int, tag: *const c_char, text: *const c_char) -> c_int;
+    }
+
+    fn redirect_stdout_to_logcat() {
```
Comment (johan-bjareholt) [inline src/android.rs]: Consider splitting android.rs into a android folder and keep this copy-pasted utilities code in a separate source file.

**Comment by ErikBjare** [comment]
I'm going to merge this now and branch out again for new stuff.

**Pull request merged**